src/Controller/ContactController.php

```diff
@@ -60,14 +60,24 @@ public function add()
         if ($_SERVER['REQUEST_METHOD'] === 'POST') {
             $userMessage = array_map('trim', $_POST);
             $this->userContactFormVerification($userMessage);
-
-            if (!empty($this->errors)) {
-                return $this->twig->render('Contact/contact.html.twig', ['errors' => $this->errors]);
+            if (isset($_POST['g-recaptcha-response']) && !empty($_POST['g-recaptcha-response'])) {
+                $secret = '6LfaewIjAAAAAGE2jVosKMD84I94tiN060GEBmhP';
```

Comment:
Yeah, now it's time to hide this in your config file

Reply:
secret key google captcha move on config file